### PR TITLE
use the same annotation syntax everywhere

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: en/reference/annotations-mapping.rst [hunk 1]
```diff
@@ -74,33 +74,35 @@ Minimal example:
 
 .. code-block:: php
 
-   <?php
```
Comment (alcaeus) [inline en/reference/annotations-mapping.rst]: Commenting once for many instances: some code blocks have an opening tag, others don't. Do you mind taking a look at that as well?

File: en/reference/introduction.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ Building the model
 ------------------
 
 Models are plain PHP classes. Note that you have several ways to define the mapping.
-For easy readability, we use the annotation mapping in this tutorial::
+For easy readability, we use the annotation mapping with PHPCR namespace in this tutorial::
```
Comment (alcaeus) [inline en/reference/introduction.rst]: Could you fix the double colon at the end?

File: en/reference/annotations-mapping.rst (hunk 1 above)
Comment (dbu) [reply en/reference/annotations-mapping.rst]: my logic was: when its code that would work on its own (e.g. a class) i put the opening tag, but when its just a fragment like a property declaration, i remove it. does that make sense? or would you simply remove it everywhere?

File: en/reference/introduction.rst (hunk 2 above)
Comment (dbu) [reply en/reference/introduction.rst]: the double colon is intentional. in re-structured text its the shortcut to tell that a php code segment follows.

File: en/reference/annotations-mapping.rst (hunk 1 above)
Comment (dbu) [reply en/reference/annotations-mapping.rst]: oh, but around here, i have been super not consistent with my own plan. i think i will just remove all of the `<?php`

Comment (alcaeus) [reply en/reference/annotations-mapping.rst]: Your reasoning makes perfect sense 👍

**Comment by dbu** [comment]
thanks for the feedback, appreciate it.

**Comment by dbu** [comment]
@dantleech do you remember how i can re-regenerate query-builder-reference file from the code? (and adjust the generator to follow our cleaned up syntax for rst)?

**Comment by dbu** [comment]
yay, dan found how we do it: a command in the phpcr-odm library. updating that in https://github.com/doctrine/phpcr-odm/pull/762 and committed the updated version here.

**Pull request merged**